types/fast-diff/index.d.ts

```diff
@@ -0,0 +1,16 @@
+// Type definitions for fast-diff 1.1
+// Project: https://github.com/jhchen/fast-diff#readme
+// Definitions by: John Renner <https://github.com/djrenren>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function diff(text1: string, text2: string, cursorPos?: number): diff.Diff[];
+
+declare namespace diff {
+    type Diff = [-1 | 0 | 1, string];
+
+    const DIFF_DELETE: -1;
```

Comment:
According to [the source code](https://github.com/jhchen/fast-diff/blob/master/diff.js#L573) and [the readme](https://github.com/jhchen/fast-diff/blob/master/README.md), these names are wrong. They should just be `DELETE`, `INSERT` and `EQUAL`, respectively.